packages/zero-schema/src/builder/table-builder.ts

```diff
@@ -43,6 +43,22 @@ export function json<T extends ReadonlyJSONValue = ReadonlyJSONValue>() {
   });
 }
 
+export function date<T extends ReadonlyJSONValue = ReadonlyJSONValue>() {
+  return new ColumnBuilder({
+    type: 'date',
+    optional: false,
+    customType: null as unknown as T,
+  });
+}
+
+export function timestamp<T extends ReadonlyJSONValue = ReadonlyJSONValue>() {
```

Comment:
`T extends number` and maybe even default it to an alias

`T extends number = Timestamp`